src/app/layout.tsx

```diff
@@ -46,10 +47,13 @@ async function getAppConfig() {
 
 export default async function RootLayout({ children }: { children: ReactNode }) {
 	const appConfig = await getAppConfig();
+	const jwt = cookies().get(EGO_JWT_KEY)?.toString();
```

Comment:
At the risk of stating what's been mentioned elsewhere (I don't see the comment on this line :shrug:):
anything to do with JWT management should go in an auth context. it doesn't seem to be needed at the layout level.